src/components/Span.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const StyledSpan = styled.span`
```

Comment:
`StyledHeadingSpan`. `StyledSpan` is somewhat opaque since `<span />` tags are applicable to a wide range of components.